content/articles/getting-started-ssl-certificates.markdown

```diff
@@ -20,14 +20,14 @@ Getting a new SSL certificate is a multi-step process and involves several parti
 
 ## SSL certificate requirements {#requirements}
 
-Before starting, take a moment to review how you will use the certificate and be sure to have access to the server configuration or a feature to install a custom certificate on your server.
+Before starting, take a moment to review how you will use the certificate and be sure to have access to the server configuration, or a feature to install a custom certificate on your server.
```

Comment:
Yes, purchasing a cert when you can't use it (e.g. GitHub Pages, or other services that don't allow you to install third party certs)